cli/build_tx.go

```diff
@@ -118,11 +119,12 @@ func (cli *CardanoCLI) buildTx(b txbuilder.TxBuilder, temp *TempManager) []strin
 			args = append(args, "--mint-execution-units", buildExUnits(mint.ExCPU, mint.ExMem))
 		}
 	}
+	mintScriptFilePaths := make([]string, 0)
```

Comment:
mintScriptFilePaths => nativeScriptFilePaths
also use `map[string]struct{}` because it's more efficient and we don't need to write a string util